modules/administration-guide/partials/proc_removing-initial-openshift-user.adoc

```diff
@@ -15,6 +15,6 @@ The following procedure describes how to remove OpenShift initial OAuth user pro
 +
 [subs="+quotes,+attributes"]
 ----
-$ oc patch checluster {prod-checluster} -n {prod-namespace} --type=json -p \
+$ oc patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
```

Comment:
```suggestion
$ {orch-cli} patch checluster/{prod-checluster} -n {prod-namespace} --type=json -p \
```

Reply:
This is a pure OpenShift procedure. Having OC installed is also a prerequisite. Even though it would work with kubectl, it is right to keep the OC there, since the action takes its place on OCP.